ingestion-edge/ingestion_edge/config.py

```diff
@@ -62,6 +62,7 @@ class Route:
     "X-Pipeline-Proxy",
     "X-Debug-ID",
     "X-Source-Tags",
+    "X-Telemetry-Agent",
```

Comment:
We don't set this in ops config so a redeploy to pick up the code change should suffice.